packages/itwin/measure-tools/src/ui-2.0/MeasureToolsUiProvider.tsx

```diff
@@ -70,7 +71,7 @@ export class MeasureToolsUiItemsProvider implements UiItemsProvider {
       if (!featureFlags?.hideAngleTool) {
         tools.push(MeasureToolDefinitions.measureAngleToolCommand);
       }
-      if (!featureFlags?.hidePerpendicularTool) {
+      if (!featureFlags?.hidePerpendicularTool && !isDrawingViewActive()) {
```

Comment:
This code isn't reactive to view change events. You need to make that tool hide itself when a 2d view is active

Reply:
Pushed the fix for this.